src/lib/services/api-token-service.ts

```diff
@@ -52,10 +53,16 @@ export class ApiTokenService {
 
     private timer: NodeJS.Timeout;
 
+    private seenTimer: NodeJS.Timeout;
```

Comment:
We could have made the precious timer in to an array, we only need to keep the reference to be able to clear them all.

Reply:
Hmm I'm not sure I understood this one, could you please clarify? 
I guess they could be a `NodeJS.Timeout[]` but if we want a single reference to each we would store them in `[0]` and `[1]` I guess? 🤔 Besides one of them is an interval and the other a timeout. 
Seems like decreased readability without much to gain, so I'm probably missing something.